### DISCUSSION TITLE
Update Album Scroll Position

### DISCUSSION THREAD

**Discussion by tinohager** [LycheeOrg/Lychee#2720] [Ideas]
### Enhancement

If I am in an album in photo viewing mode and click on 40 photos further, it would be great if I switch back to the album photo overview and the scroll position changes to this point so that I could then re-enter at this point. Especially for albums with a lot of photos this makes perfect sense.

**Comment by ildyria** [comment]
Isn't this already taken care of: #2694

**Comment by tinohager** [comment]
in any case, it does not yet work in my dev instance

**Comment by tinohager** [comment]
@ildyria I have tested it again with the latest version but the scroll position is not adjusted
